source/mobile/mobile-faq.rst

```diff
@@ -157,19 +157,19 @@ All files in the ``/assets/base`` folder can be overriden as needed without conf
 2. Copy any files or folders that you wish to replace from ``/assets/base`` into ``/assets/override``.
 3. Make your changes to the files in ``/assets/override``.
 
-When you compile the app or run ``make dist/assets``, the contents of those two folders will be merged with files in ``/assets/override``, taking precedence in the case of any conflicts. For binary files such as images, an overridden file will completely replace the base version, while json files will be merged so that fields not set in the overridden copy use the base version.
+When you compile the app or run ``make dist/assets``, the contents of those two folders will be merged with files in ``/assets/override``, taking precedence in the case of any conflicts. For binary files such as images, an overridden file will completely replace the base version, while ``JSON`` files will be merged so that fields not set in the overridden copy use the base version.
 
 For a more specific example of how to use this feature, see the following section.
```

Comment:
@esethna, do you know which doc/section this should link to? Or should we remove this line?

Reply:
all this info is in the developers doc if I'm not mistaken

Reply:
Thanks, @enahum! I've taken a look through here: https://developers.mattermost.com/contribute/mobile/ but couldn't pin down the exact section (maybe wasn't looking hard enough). Is a general link sufficient?